packages/react-devtools-scheduling-profiler/src/CanvasPage.js

```diff
@@ -461,14 +481,7 @@ function AutoSizedCanvas({
       userTimingMarksView.onHover = userTimingMark => {
         if (!hoveredEvent || hoveredEvent.userTimingMark !== userTimingMark) {
           setHoveredEvent({
-            componentMeasure: null,
-            flamechartStackFrame: null,
-            measure: null,
-            nativeEvent: null,
-            networkMeasure: null,
-            schedulingEvent: null,
-            snapshot: null,
-            suspenseEvent: null,
+            ...EMPTY_CONTEXT_INFO,
```

Comment:
All of the repetition in this file was driving me crazy so I cleaned it up a bit with this change.